apiron/endpoint/stub_endpoint.py

```diff
@@ -0,0 +1,18 @@
+class StubEndpoint:
+    """
+    A stub endpoint designed to return a pre-baked response
+
+    The intent is to allow for a service to be implemented
+    before the endpoint is complete.
+    """
+
+    def __init__(self, stub_response=None, **kwargs):
+        """
+        :param stub_response:
+            A pre-baked response, like 'stub response' or {'stub': 'response'}
```

Comment:
Now would be a good time to wrap these in backticks! On the next line, `**kwargs` should be `\**kwargs` to suppress a Sphinx warning (seems to have the desired outcome, either way...)